CDCgov/prime-reportstream
# 8180 Convert SR FHIR samples

This PR updates some configs to support SR data, as well as adding a new config specific to SR.

Test Steps:
1. Check out this branch.
2. Run `./gradlew primecli --args="fhirdata -i <sample file> --output-format HL7 -s metadata/hl7_mapping/Senders/SimpleReport.yml"`, where <sample file> is one of the SR sample files attached here.
3. Verify that HL7 v2 data is generated and that the FHIR data is present in the HL7 v2 message.

No PII samples:
[bundle2.fhir](https://github.com/CDCgov/prime-reportstream/files/10698380/bundle2.txt)
[bundle3.fhir](https://github.com/CDCgov/prime-reportstream/files/10698381/bundle3.txt)
[bundle1.fhir](https://github.com/CDCgov/prime-reportstream/files/10698382/bundle1.txt)

## Changes
- Updated base config with some default values
- Added a missing email element
- Added SR config

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
